packages/react/src/AvatarStack/AvatarStack.tsx

```diff
@@ -195,6 +203,9 @@ const AvatarStack = ({
   className,
   sx: sxProp = defaultSxProp,
 }: AvatarStackProps) => {
+  const [hasInteractiveChildren, setHasInteractiveChildren] = useState<boolean | undefined>(false)
+  const AvatarStackContainer = useRef(null)
```

Comment:
style nit (and definitely non blocking lol) we could use lowercase for `useRef` here since we typically only use pascal case for component names or modules

```suggestion
  const avatarStackContainer = useRef(null)
```

Reply:
Didn't realize I was using pascal case 🤦 Should be updated now!